mechanics.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/python3
+import xml.etree.ElementTree as xml
+import os
+import utils
+
+def getKeywords(rawFolder, locale):
+    TOOLTIP_STRINGS_PATH = rawFolder + "tooltips_" + locale + ".csv"
```

Comment:
methods like this should receive parsed csv data, instead of reading from the filesystem.
Limit filesystem access to an isolated area